core/domain/feedback_jobs_continuous.py

```diff
@@ -44,15 +46,33 @@ class FeedbackAnalyticsAggregator(jobs.BaseContinuousComputationManager):
 
     @classmethod
     def get_event_types_listened_to(cls):
+        """Get the event types that this class is subscribed to.
+
+        Returns:
+            list(str). List of event types that this class is subscribed to.
+        """
         return [feconf.EVENT_TYPE_NEW_THREAD_CREATED,
                 feconf.EVENT_TYPE_THREAD_STATUS_CHANGED]
 
     @classmethod
     def _get_realtime_datastore_class(cls):
+        """Get the realtime datastore class used by the realtime layer.
+
+        Returns:
+            FeedbackAnalyticsRealtimeModel. Datastore class used by the
+            realtime layer, which should be a subclass of
+            BaseRealtimeDatastoreClassForContinuousComputations.
+        """
         return FeedbackAnalyticsRealtimeModel
 
     @classmethod
     def _get_batch_job_manager_class(cls):
+        """Get manager class for the continuously-running batch job.
+
+        Returns:
+            FeedbackAnalyticsMRJobManager. Manager class for continuous-running
```

Comment:
Done

Reply:
Um, I think this might have been misinterpreted. `type()` is a Python function: see https://docs.python.org/2/library/functions.html#type. E.g. if you do `type('abc')` you'll get str, so the typeinfo in the docstring should be "str". Could you do the same thing for FeedbackAnalyticsMRJobManager?

Reply:
Done